src/lib/components/colors/FormatGroup.svelte

```diff
@@ -0,0 +1,60 @@
+<script lang="ts">
+  import { inGamut } from 'colorjs.io/fn';
+  import type { PlainColorObject } from 'colorjs.io/types/src/color';
+
+  import Output from '$lib/components/colors/Output.svelte';
+  import type { FormatGroup } from '$lib/constants';
+  import { ColorSpace } from '$lib/stores';
+  import { getSpaceFromFormatId } from '$lib/utils';
+
+  import ExternalLink from '../util/ExternalLink.svelte';
+
+  export let type: 'bg' | 'fg';
+  export let color: PlainColorObject;
+  export let formatGroup: FormatGroup;
+
+  function inGamutForSpace(color: PlainColorObject) {
+    if (!formatGroup.gamutFormat) return true;
+    const gamutSpace = ColorSpace.get(
+      getSpaceFromFormatId(formatGroup.gamutFormat),
+    );
+    return inGamut(color, gamutSpace);
+  }
+  $: isInGamut = inGamutForSpace(color);
+</script>
+
+<div data-content="format-group">
+  <div class="format-group-heading">
+    <h2 class="label section-heading">{formatGroup.name}</h2>
+    {#if !isInGamut}
+      <span data-color-info="warning"
+        >Selected color is <ExternalLink
+          href="https://www.w3.org/TR/css-color-4/#out-of-gamut"
+          >outside the {formatGroup.gamutName} gamut.</ExternalLink
+        ></span
+      >
+    {/if}
+  </div>
+  {#each formatGroup.formats as format (format)}
+    <Output {type} {color} {format} />
+  {/each}
+</div>
+
+<style lang="scss">
+  [data-content~='format-group'] {
+    --heading-transform: none;
+
+    column-gap: var(--gutter);
+    display: grid;
+    grid-template-columns: auto 1fr;
```

Comment:
What is the purpose of this grid? It seems like nothing is ever in the second column?

![Screenshot 2023-11-22 at 9 32 27 AM](https://github.com/oddbird/oddcontrast/assets/104161/2599e17a-cbd6-4b9b-b059-8eaacc5613a7)

Reply:
fixed in https://github.com/oddbird/oddcontrast/pull/137/commits/8b130b1374a4394016de38c3d551a172a7842d0b